### PR TITLE
Add a quick import progress

### LINKED ISSUE #585: Progress bar for quick import
It might be nice to have an overall progress bar at the top of the screen while the "quick import" is going. That way, the user knows that there is a process going automatically so they know what is happening.

### PR DESCRIPTION
**Pull request by bruyeret**
It takes all steps into account and uses a weight per step to account for the inequal time taken per step

Close #585

### CODE REVIEW

**Review by arjunrajlab — CHANGES_REQUESTED** [review]
Progress bar looks great! Perhaps could make it a bit thicker. I also think somehow dimming out the interface underneath would be good, and perhaps hiding the buttons. That would be a bit less confusing for the user, who might want to click on the buttons while the quick import is proceeding.

**Comment by luciemac** [comment]
LGTM

**Comment by bruyeret** [comment]
Is it better now @arjunrajlab ?

**Review by arjunrajlab — APPROVED** [review]
Yes, looks great!

**Comment by bruyeret** [comment]
Do you want to review the two last commits @luciemac ?

File: src/components/GirderLocationChooser.vue [hunk 1]
```diff
@@ -12,6 +12,7 @@
           class="pl-4"
           root-location-disabled
           :location="selected"
+          :readonly="true"
```
Comment (luciemac) [inline src/components/GirderLocationChooser.vue:15]: can't you go with only "readonly" property instead of ":readonly="true""?

**Comment by luciemac** [comment]
then, LGTM

File: src/components/GirderLocationChooser.vue (hunk 1 above)
Comment (bruyeret) [reply src/components/GirderLocationChooser.vue]: Yes you're right!

**Pull request merged**